### PR TITLE
BER Dany & Dioni CLUE Done

### PR DESCRIPTION
**Pull request by dionifss**

### CODE REVIEW

File: src/clue.js [hunk 1]
```diff
@@ -1,14 +1,128 @@
...
+  { name: "trophy", weight: 25 },
+  { name: "pistol", weight: 20 }
+];
+
+//ITERATION 2
+
+function selectRandom(array) {
+  random = Math.floor(Math.random() * array.length);
+  return array[random];
+}
```
Comment (fonsogms) [inline src/clue.js]: awesome

File: src/clue.js [hunk 2]
```diff
@@ -1,14 +1,128 @@
...
+
+function pickMystery() {
+  let mystery = {
+    room: selectRandom(roomsArray),
+    weapon: selectRandom(weaponsArray),
+    suspect: selectRandom(suspectsArray)
+  };
+  return mystery;
```
Comment (fonsogms) [inline src/clue.js]: god damn it, well done guys

File: src/clue.js [hunk 3]
```diff
@@ -1,14 +1,128 @@
...
+  return (
+    ar.suspect.firstName +
+    " " +
+    ar.suspect.lastName +
+    " killed Mr. Boddy using the " +
+    ar.weapon.name +
+    " in the " +
+    ar.room.name +
+    "!"
+  );
```
Comment (fonsogms) [inline src/clue.js]: all cool in the school

**Comment by dionifss** [comment]
Thanks mexxx ;)


Dionisio Ugalde
www.balonmanoproshop.com
+34 659314017

**Pull request closed without merging**